storageprovider/csp/container_storage_provider.go

```diff
@@ -936,12 +936,12 @@ func getCspClient(credentials *storageprovider.Credentials) (*connectivity.Clien
 	// Off-Array CSP
 	cspURI := fmt.Sprintf("http://%s:%d", credentials.ServiceName, credentials.ServicePort)
 
-	log.Tracef(">>>>> getCspClient (service) using URI %s and username %s", cspURI, credentials.Username)
+	log.Tracef(">>>>> getCspClient (service) using URI %s and username %s with timeout set to %d seconds", cspURI, credentials.Username, credentials.CspClientTimeout)
 	defer log.Trace("<<<<< getCspClient")
 
+	cspHTTPClientTimeout := time.Duration(credentials.CspClientTimeout)*time.Second
```

Comment:
the CspClientTimeout could be empty. Please check and set it to default if missing.

Reply:
@raunakkumar It will not be empty due to the default value set here https://github.com/hpe-storage/csi-driver/pull/271/files#diff-0a8bad30eb3609c02feaa2922f850d1f8daefcad3765d4c10d91e2d02eb5da63R81
and here https://github.com/hpe-storage/csi-driver/pull/271/files#diff-e8a7e777d80a14b455bdbf7aae3f28ad8082ffa0a06579e11cc1af741b5f98f7R359

Reply:
getCspClient could be invoked from elsewhere too.

Reply:
Done https://github.com/hpe-storage/common-host-libs/pull/149/files#diff-b607804061eca187a03afd96d30ebeb0169f4183490ee5f54cbb5077ab555838R939